src/lib/components/EmptyResults/EmptyResults.js

```diff
@@ -35,6 +36,7 @@ class EmptyResults extends Component {
         <Element
           queryString={queryString}
           resetQuery={this.resetQuery}
+          extraContent={extraContent}
```

Comment:
Where is this used? It's a property specific to `Overridable`?

Reply:
no, it is if you want to pass something additional to the EmptyResult, f.e. button to add a new instance or more information. It is more flexible this way than to override the whole component just for this.

Reply:
But is not used in the `Element`, no?

Reply:
no it's not used, but I think it should be passed as a prop

Reply:
Ok I missed the fact that the Element is overridable :) Then, it looks good!